imports/plugins/core/orders/server/no-meteor/mutations/createOrder.js

```diff
@@ -261,9 +263,11 @@ export default async function createOrder(context, input) {
       workflow: { status: "new", workflow: ["coreOrderWorkflow/notStarted"] }
     };
 
+    const cart = await getCartById(context, cartId, { cartToken, throwIfNotFound: true });
+    const cartWithSummary = await xformCartCheckout(collections, cart);
     // Verify that the price for the chosen shipment method on each group matches between what the client
     // provided and what the current quote is.
-    const rates = await context.queries.getFulfillmentMethodsWithQuotes(finalGroup, context);
+    const rates = await context.queries.getFulfillmentMethodsWithQuotes(finalGroup, cartWithSummary, context);
```

Comment:
Point 1 above says that `createOrder` shouldn't know anything about carts, however `cartId` is passed into `xFormOrderGroupToCommonOrder`. Is it OK since it's just the ID and not the cart itself? It's also not manipulated or used in `xFormOrderGroupToCommonOrder`, it just gets passed through it.